app/routes/projects/index.tsx

```diff
@@ -439,7 +439,7 @@ export default function Projects() {
                     aria-controls="panel2a-controls"
                     id="panel2a-header"
                   >
-                    <strong>Roles</strong>
+                    <strong>Has Roles</strong>
```

Comment:
I think this label and the next one are more descriptive. What do you think?

Reply:
Maybe Actual Roles